app/views/admin/workload-points/workload-points.html

```diff
@@ -0,0 +1,307 @@
+                      {% else %}
+                        {{ custody.cusc1 }}
+                      {% endif %}
+                    </td>
+                  </tr>
+                  <tr>
+                    <th scope="row">Cus B2</th>
+                    <td class="numeric">
+                      {% if edit %}
+                        <input class="form-control numeric" name="cusb2" type="text" style="width: 50px" value="{{custody.cusb2}}">
+                      {% else %}
+                        {{ custody.cusb2 }}
+                      {% endif %}
+                    </td>
+                  </tr>
+                  <tr>
+                    <th scope="row">Cus B1</th>
+                    <td class="numeric">
+                      {% if edit %}
+                        <input class="form-control numeric" name="cusb1" type="text" style="width: 50px" value="{{custody.cusb1}}">
+                      {% else %}
+                        {{ custody.cusb1 }}
+                      {% endif %}
+                    </td>
+                  </tr>
+                  <tr>
+                    <th scope="row">Cus A</th>
+                    <td class="numeric">
+                      {% if edit %}
+                        <input class="form-control numeric" name="cusa" type="text" style="width: 50px" value="{{custody.cusa}}">
+                      {% else %}
+                        {{ custody.cusa }}
+                      {% endif %}
+                    </td>
+                  </tr>
+                </div>
+              </tbody>
+            </table>
+          </div>
+        </div>
+      </div>
+
+      <div id="license">
+        <div class="grid-row">
+          <div class="column-two-thirds">
+            <table>
+              <tbody>
+                <tr>
+                  <th scope="row">Lic D2</th>
+                  <td class="numeric">2</td>
+                </tr>
+                <tr>
+                  <th scope="row">Lic D1</th>
+                  <td class="numeric">1</td>
+                </tr>
+                <tr>
+                  <th scope="row">Lic C2</th>
+                  <td class="numeric">2</td>
+                </tr>
+                <tr>
+                  <th scope="row">Lic C1</th>
+                  <td class="numeric">2</td>
+                </tr>
+                <tr>
+                  <th scope="row">Lic B2</th>
+                  <td class="numeric">4</td>
+                </tr>
+                <tr>
+                  <th scope="row">Lic B1</th>
+                  <td class="numeric">4</td>
+                </tr>
+                <tr>
+                  <th scope="row">Lic A</th>
+                  <td class="numeric">5</td>
+                </tr>
+              </tbody>
+            </table>
+          </div>
+        </div>
+      </div>
+      
+      <div id="community">
```

Comment:
Maybe in the real app, but not sure it's worth it for the prototype. We don't apply the same structure in our app anyway